plio/io/io_controlnetwork.py

```diff
@@ -29,6 +32,71 @@ def write_filelist(lst, path="fromlist.lis"):
         handle.write('\n')
     return
 
+
+class MessageType(IntEnum):
+    """
+    An enum to mirror the ISIS3 MeasureLogData enum.
+    """
+    GoodnessOfFit = 2
+    MinimumPixelZScore = 3
+    MaximumPixelZScore = 4
+    PixelShift = 5
+    WholePixelCorrelation = 6
+    SubPixelCorrelation = 7 
+
+class Log():
```

Comment:
Similar comment to the one above, this is checking the MeasureMessageType, so this should probably be names 'MessageLog'